src/backend/api/Fusion.Resources.Logic/Requests/Commands/ResourceAllocationRequest/ResourceOwner/ProvisionResourceOwnerRequest.cs

```diff
@@ -0,0 +1,325 @@
+﻿using Fusion.ApiClients.Org;
+using Fusion.Resources.Database;
+using Fusion.Resources.Database.Entities;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Logic.Commands
+{
+    public partial class ResourceAllocationRequest
+    {
+        public partial class ResourceOwner
+        {
+
+            internal class ProvisionResourceOwnerRequest : IRequest
+            {
+                public ProvisionResourceOwnerRequest(Guid requestId)
+                {
+                    RequestId = requestId;
+                }
+
+                public Guid RequestId { get; }
+
+
+                public class Handler : AsyncRequestHandler<ProvisionResourceOwnerRequest>
+                {
+                    private readonly IOrgApiClient client;
+                    private readonly ResourcesDbContext resourcesDb;
+
+                    public Handler(ResourcesDbContext resourcesDb, IOrgApiClientFactory orgApiClientFactory)
+                    {
+                        this.client = orgApiClientFactory.CreateClient(ApiClientMode.Application);
+                        this.resourcesDb = resourcesDb;
+                    }
+
+                    protected override async Task Handle(ProvisionResourceOwnerRequest request, CancellationToken cancellationToken)
+                    {
+                        var dbRequest = await resourcesDb.ResourceAllocationRequests
+                            .Include(r => r.Project)
+                            .FirstOrDefaultAsync(r => r.Id == request.RequestId);
+
+                        await ProvisionAsync(dbRequest);
+                    }
+
+                    private async Task ProvisionAsync(DbResourceAllocationRequest dbRequest)
+                    {
+                        // Need to get the raw position, in case model has changed.. When putting position, fields not 
+                        var rawPosition = await client.GetAsync<JObject>($"/projects/{dbRequest.Project.OrgProjectId}/positions/{dbRequest.OrgPositionId}?api-version=2.0");
```

Comment:
Not sure how that WCF "model" would provide any improvement over just updating those properties we care about? This pattern was developed for the pims > org sync and has been running without any issue overwriting existing data the sync does not know about.

I would prefer having weeks to write this integration logic, however with a few hours a day, this will have to do :) 

Was thinking that this could be something to put in the api client library. However the time is not on our side here, and with parameters rapidly chaning it would result in a much larger feedback loop for small and/or major tweeks/updates.

I think when we have started testing the flow, and discovered most of the "breaking" changes to be made, we can refactor this.